dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -39,7 +39,9 @@ startsecs=0
 command=/usr/sbin/fancontrol
 priority=4
 autostart=false
-autorestart=false
+autorestart=unexpected
+startsecs=10
```

Comment:
10 seconds is the default value for `stopwaitsecs` in supervisor, thus this line is unnecessary. Also, what does `stopwaitsecs` have to do with restarting the process?

Reply:
Removed